app/Console/Commands/CreateMasterVoucherLogReport.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Carbon\Carbon;
+use DB;
+use Excel;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Console\Command;
+use Log;
+use Maatwebsite\Excel\Writers\LaravelExcelWriter;
```

Comment:
Nice - is this the replacement for deprecated phpexcel package we used before?

Reply:
We really should upgrade. Even 5.4 been out of security fixes for a year already. https://laravel-news.com/laravel-release-process